root/report/not_various_artists.tt

```diff
@@ -0,0 +1,15 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings incorrectly linked to Various Artists') full_width=1 -%]
+
+<h1>[% l('Recordings incorrectly linked to Various Artists') %]</h1>
+
+<ul>
+    <li>[% l('This report shows recordings linked to Various Artists entity
```

Comment:
"linked to the"

For the line below I think I'd say `without "Various Artists" as the credited name.` since it's still the artist credit, just using a different name.